<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategorySetupPage</class>
 <widget class="QWizardPage" name="CategorySetupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Category Setup</string>
  </property>
  <property name="subTitle">
   <string>Choose which tests you would like to run, and which test results you would like to use to assign categories to each line.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="frequencyWindowLabel">
       <property name="text">
        <string>Frequency window</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="frequencyWindowSpinBox">
       <property name="toolTip">
        <string>If a line is more than this far from the cavity frequency, it will not be categorized.</string>
       </property>
       <property name="suffix">
        <string> kHz</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>50</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="testsBox">
       <property name="title">
        <string>Tests</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="dipoleTestBox">
          <property name="text">
           <string>Dipole Moment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="dcTestBox">
          <property name="text">
           <string>Discharge On/Off</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="voltageTestBox">
          <property name="text">
           <string>Discharge Voltage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="magnetTestBox">
          <property name="text">
           <string>Magnet On/Off</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="categoriesBox">
       <property name="title">
        <string>Categorize by...</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="dipoleCategoryBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Dipole Moment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="dcCategoryBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Discharge On/Off</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="voltageCategoryBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Discharge Voltage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="magnetCategoryBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Magnet On/Off</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="dipolesBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Dipoles</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListView" name="dipolesView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="dipoleAddButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="dipoleRemoveButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="voltagesBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Voltages</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListView" name="voltagesView">
          <property name="spacing">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="voltageAddButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="voltageRemoveButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/remove.png</normaloff>:/icons/remove.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
